source/docs/software/wpilib-tools/smartdashboard/test-mode-and-live-window/displaying-LiveWindow-values.rst

```diff
@@ -1,13 +1,51 @@
-Displaying LiveWindow values
+Displaying LiveWindow Values
 ============================
 
-Typically LiveWindows are displayed as part of the automatically generated RobotBuilder code. You may also display LiveWindow values by writing the code yourself and adding it to your robot program. LiveWindow will display values grouped in subsystems. This is a convenient method of displaying whether they are actual command based program subsystems or just a grouping that you decide to use in your program.
+.. note::
```

Comment:
This formatting seems weird. Can you put the note on the same line?

Reply:
I was thinking that the note was useful because this code is largely unnecessary.

Having it integrated with the paragraph means it blends in more and teams are more likely to jump to the code first.